backend/routes/PostReaction/AttendeeHelperFunction.py

```
@@ -0,0 +1,59 @@
+from fastapi import APIRouter, Depends, HTTPException, Query
```

Ruff (0.8.2)

`fastapi.APIRouter` imported but unused

Remove unused import

(F401)